vtStor.ATest/vtStor.ATest.pro

```diff
@@ -0,0 +1,37 @@
+QT       += core
+
+QT       -= gui
+
+TARGET = vtStor.ATest
+CONFIG   += console
+CONFIG   -= app_bundle
+QMAKE_CXXFLAGS += -std=c++14
+QMAKE_CXXFLAGS += -fpermissive
+
+TEMPLATE = app
+
+
+SOURCES += main.cpp
+
+win32:CONFIG(release, debug|release): LIBS += -L$$PWD/../vtStor-build-desktop-Qt_4_8_1_in_PATH__System__Release/release/ -lvtStor
+else:win32:CONFIG(debug, debug|release): LIBS += -L$$PWD/../vtStor-build-desktop-Qt_4_8_1_in_PATH__System__Release/debug/ -lvtStor
+else:unix:!macx:!symbian: LIBS += -L$$PWD/../vtStor-build-desktop-Qt_4_8_1_in_PATH__System__Release/ -lvtStor
+
+INCLUDEPATH += $$PWD/../vtStor-build-desktop-Qt_4_8_1_in_PATH__System__Release
+DEPENDPATH += $$PWD/../vtStor-build-desktop-Qt_4_8_1_in_PATH__System__Release
+
+win32:CONFIG(release, debug|release): LIBS += -L$$PWD/../vtStorAtaProtocol-build-desktop-Qt_4_8_1_in_PATH__System__Release/release/ -lvtStorAtaProtocol
```

Comment:
Hi a.Hai, the path is generated by IDE. Name of folder base on convention of QT. It easy to understand (version IDE, mode build - Release or Debug)